gamemodes/terrortown/gamemode/shared/sh_equip_items.lua

```diff
@@ -56,20 +56,13 @@ EquipmentItems = EquipmentItems or setmetatable(
 )
 
 ---
--- Returns the equipment base table merged with the given data
--- @param table data
--- @param table eq
--- @return table
--- @todo improve description
+-- Adds all needed parameters for TTT2
+-- @param string name The classname this equipment shall have
+-- @param table eq Equipment, that you add the KeyValues to
 -- @internal
 -- @realm shared
-function GetEquipmentBase(data, eq)
-	if not eq or eq.inited then
-		return eq
-	end
-
-	local name = WEPS.GetClass(eq)
-	if not name then return end
+function AddStandardKeyValues(eq, name)
```

Comment:
This method name should contain something equipment related (otherwise this sounds too generic but is actually specific to be used with equipment only)

(ideally those functions would be transformed into a module or something like that, but that might be out of the scope of this PR)

Reply:
I agree and changed the name, in general your trello mentioned, that sh_equip_items needs a refactor. With that, I atleast made some functions more clear in purpose. But the overloading of functions doing mutliple stuff at once is a bit weird. There is a base getter, that assigns the keyvalues, which is really weird. Didnt want to break anything, so I left it there. In general the getter shouldnt be abused to assign values to uninitialized equipment......